docs/sla/index.mdx

```diff
@@ -30,6 +30,17 @@ The following policy applies to both our cloud-based (managed instance) and on-p
 
 >NOTE: Premium support with enhanced SLAs can be added to your Enterprise plans as an add-on. Our business hours, defined as Sunday 2 PM PST to Friday 5 PM PST, align with our 24x5 support coverage.
 
+### For Cody Pro plans
+
+| Severity level |                                                                     Description                                                                      |                    Response time                    | Support availability |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------- | -------------------- |
+| 0              | Emergency: Total loss of service or security-related issue                                                                           | Within one business day of identifying the issue | 8am to 5pm PST (Monday-Friday) |
+| 1              | Severe impact: Billing issues, login issues, refunds | Within one business day of identifying the issue  |8am to 5pm PST (Monday-Friday) |
```

Comment:
```suggestion
| 1              | Severe impact: Billing issues, login issues, refunds | Within one business day of identifying the issue  |8 AM to 5 PM PST (Monday-Friday) |
```

Reply:
Let's remove "refunds" from this, we don't offer refunds. We can offer free months of Cody Pro but I don't think we should publicly make that known.